addons/web/static/src/views/list/list_renderer.js

```diff
@@ -162,13 +163,16 @@ export class ListRenderer extends Component {
         if (field.sortable) {
             classNames.push("o_column_sortable");
         }
-        const orderedBy = this.props.list.orderedBy || {};
-        if (column.name === orderedBy.name) {
+        const orderedBy = this.props.list.orderedBy;
```

Comment:
```suggestion
        const orderBy = this.props.list.orderedBy;
```
We'll rename that in datapoints, so maybe already use the correct name here :-)
(orderBy would be more consistent with groupBy)